### ISSUE TITLE
Changes in ref: dev

### ISSUE THREAD

**Issue by greedyRoach** [aleksanderwozniak/table_calendar#223]
I'm currently using this in my flutter pubspec
```
table_calendar:
    git:
      url: git://github.com/aleksanderwozniak/table_calendar.git
      ref: dev
```
Because i need to use
```
calendarController.events = events;
```
But i'm getting a lot of errors:
<img width="604" alt="Captura de Pantalla 2020-03-27 a la(s) 13 01 33" src="https://user-images.githubusercontent.com/57685250/77790903-3dbe2e80-702b-11ea-9e70-203c39359e2e.png">

It used to work fine, but then i changed my pub to:
```
table_calendar: ^2.2.3
```
To check new features (i used to have an older version of the package). Then i reverted the pubspec to:
```
table_calendar:
    git:
      url: git://github.com/aleksanderwozniak/table_calendar.git
      ref: dev
```
Because as i said earlier i need to use:
```
calendarController.events = events;
```
but now is all messed up, what have i done ?:(

in summary, i did this:

1. I used to have an old version of the package
2. Then changed to ref:dev so i could set events in the calendarController
2. Changed pubspec to point to the most recent version of the package (2.2.3)
3. I reverted to refer to dev (ref:dev) in the pubspec again so i could set events in the calendarController
4. Now i have a bunch of errors

Have tried:
1. Flutter clean
2. Flutter package get
3. flutter pub upgrade 

but with no results

**Comment by greedyRoach** [comment]
i need to keep setting events in the calendarController, i use a custom way of setting colors in days with activities

![Simulator Screen Shot - iPhone 8 - 2020-03-18 at 14 56 49](https://user-images.githubusercontent.com/57685250/77792217-98588a00-702d-11ea-9049-d2c5efe39598.png)

**Comment by aleksanderwozniak** [comment]
You can use TableCalendar version `2.2.3` and set the events via `events` property. Whenever you need to update them, simply make changes to underlying data structure and rebuild the widget - here you can find some gists that might be helpful:

- [Dynamic events](https://gist.github.com/aleksanderwozniak/1367892cbe39c25afda403b2300652ca)
- [Adding a new event](https://gist.github.com/aleksanderwozniak/e9eb98bf28bd9fbb336039e1936c59aa)

**Comment by greedyRoach** [comment]
ok, i will checkout that option.

by the way, i don't know why it stopped working, i put my project to a previous state before updating the package and is working again

![WhatsApp Image 2020-03-27 at 2 12 11 PM](https://user-images.githubusercontent.com/57685250/77796653-52072900-7035-11ea-8d71-ef1ea11dbf1e.jpeg)

thanks aleksanderwozniak, your package is helping me tons
